nc_news frontend/src/Body/ArticleFeed.jsx

```diff
@@ -0,0 +1,25 @@
+import { useContext, useEffect } from "react";
+import { Container } from "react-bootstrap";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import { userContext } from "../contexts/userContext";
+import { fetchArticles } from "../util/api";
+import ArticleCard from "../Components/Article";
+
+export default function ArticleFeed() {
+  const { articleFeed, setArticleFeed } = useContext(userContext);
```

Comment:
useContext should only be used for data you want access to throughout your application, for example a logged in user or a dark theme switcher etc. 

I would replace this with useState instead and then pass articleFeed props to ArticleCard